js/script.js

```diff
@@ -86,14 +85,4 @@ window.addEventListener('DOMContentLoaded', () => {
     }
 
```

Comment:
Так по поводу фукнции moveSlide, выглядит топорно, еще и с этими ифами. Вот более красивый вариант 
```suggestion
function moveSlide(direction) {
    // Количество доступных слайдов для переключения
    const numSlides = cardList.length - 3; 
    slideIndex = (slideIndex + direction + numSlides) % numSlides;

    updateSlider();
}
```

Reply:
Еще надеюсь ты перепишешь логику слайдера, могу описать ниже проблемы которые стоит решить

1. Я бы сделал отдельный класс, который бы инициализировал слайдер, ты бы в него прокидывал врапер с карточками, количество карточек в ряд
2. Совет как можно на изи сделать слайдер без просчетов трансформаций - это сделать контейнер с карточками с **_горизонтальным_** скроллом и вызывать на нужной карточке scrollIntoView и тогда js сам сделает красивый слайд к карточке (Напиши в телеге, когда доберешься до этого, я мб более подробнее расскажу если не понял)
2.1 Сделав пункт 2, стоит решить проблему когда ты ресайзишь окно браузера, у тебя в теории будут сьезжать слайды в бок, что не очень красиво, желательно это пофиксить, делается это на изи